app/src/components/generic/recordCard/RecordCard.tsx

```diff
@@ -28,8 +33,7 @@ function RecordCard({
                     {heading}
                 </Card.Heading>
                 {detailsElement}
-
-                {downloadStage === DOWNLOAD_STAGE.SUCCEEDED && (
+                {downloadStage === DOWNLOAD_STAGE.SUCCEEDED && userIsGpAdmin && (
```

Comment:
Instead, we should do a check to see that the role is NOT GP clinical user.

Reply:
This has now been addressed.